tests/acceptance/test_multi_threading.py

```diff
@@ -123,7 +123,7 @@ def test___multiple_tasks___get_set_float_and_string_properties___no_errors(
                 _get_set_property_thread_main,
                 start_barrier,
                 stop_semaphore,
-                task,
+                channel,
```

Comment:
Mypy identified a bug in this test: it was getting and setting `task.ai_max` instead of `channel.ai_max`.